manual/Remove-contaminated-files-from-tdr.md

```diff
@@ -0,0 +1,168 @@
+# Remove contaminated files from the TDR system
+
+This guides the steps necessary to remove files from the TDR database and S3 buckets.
+
+## SSH tunneling to connect bastion from your local machine - (required to delete from intg/staging database)
+
+1. Install the latest aws cli on your machine.
+2. Create an AWS CLI profile
+   ```
+   aws configure sso
+   aws sso login --profile tdr-integration ( change to the profile you created)
+   ```
+3. Add your ssh key to the bastion
+    - sudo su ec2-user
+    - cd .ssh
+    - nano authorised_key
+4. On your machine add a line to your .ssh config file
```

Comment:
Can you mention explicitly the directory for the steps i.e. I think I remember we modified files in the following directory
/etc/ssh/ssh_config
/etc/hosts